### PR TITLE
DE filter

### PR DESCRIPTION
**Pull request by robinpaul85**
## Description

In this PR, now min_count and min_total_count can now be controlled through the UI. min_count is used to calculate the cpm cutoff and min_total_count is an absolute cutoff. For more details, see the filterByExpr [source code ](https://rdrr.io/bioc/edgeR/src/R/filterByExpr.R)in edgeR.

For testing, may use this sessions file
[sjmb12_DE.txt](https://github.com/stjude/proteinpaint/files/15381858/sjmb12_DE.txt)

## Checklist

[Check each task](https://github.com/stjude/proteinpaint/wiki/Pull-Request-Checklist) that has been performed or verified to be not applicable.
- [ ] Tests: added and/or passed unit and integration tests, or N/A
- [ ] Todos: commented or documented, or N/A
- [ ] Notable Changes: updated release.txt, prefixed a commit message with "fix:" or "feat:", added to an internal tracking document, or N/A

### CODE REVIEW

File: client/plots/DEanalysis.js [hunk 1]
```diff
@@ -53,6 +53,24 @@ class DEanalysis {
 	async setControls(output) {
 		this.dom.controlsDiv.selectAll('*').remove()
 		const inputs = [
+			{
+				label: 'Min count',
+				type: 'number',
+				chartType: 'DEanalysis',
+				settingsKey: 'min_count',
+				title: 'Min count',
```
Comment (xzhou82) [inline client/plots/DEanalysis.js:61]: at title, fully specify the purpose "Genes with xxx will be excluded"

Comment (robinpaul85) [reply client/plots/DEanalysis.js]: I added a title to both min_count and min_total_count.

File: client/plots/DEanalysis.js [hunk 2]
```diff
@@ -53,6 +53,25 @@ class DEanalysis {
 	async setControls(output) {
 		this.dom.controlsDiv.selectAll('*').remove()
 		const inputs = [
+			{
+				label: 'Min count',
```
Comment (xzhou82) [inline client/plots/DEanalysis.js:57]: for these two new params, will it be better to call them `Min read count per sample` and `Min total read count`
at a minimum, if the cutoff is about read count, then label must indicate so

**Comment by xzhou82** [comment]
works well.
later should investigate that on UI, after changing cutoff parameter, the app should show a cloak and indicate to user it's loading which takes seconds

File: client/plots/DEanalysis.js (hunk 2 above)
Comment (robinpaul85) [reply client/plots/DEanalysis.js:57]: Discussed privately in slack

**Review by xzhou82 — APPROVED** [review]
you should actually make `label` more intuitive. at minimum mention `read count` otherwise it's not user friendly

**Review by xzhou82 — APPROVED** [review]

**Comment by robinpaul85** [comment]
> you should actually make `label` more intuitive. at minimum mention `read count` otherwise it's not user friendly

Added `read count` to label

**Pull request merged**